### ISSUE TITLE
CLASSIFICATION OF SONAR DATA SET USING NEURAL NETWORK TRAINED BY GRAY WOLF OPTIMIZATION

### ISSUE THREAD

**Issue by koptimizer** [KGJsGit/my_PaperList#23]
## :clipboard: 논문의 정보를 알려주세요.
- CLASSIFICATION OF SONAR DATA SET USING NEURAL NETWORK TRAINED BY GRAY WOLF OPTIMIZATION
- M. R. Mosavi, M. Khishe, A. Ghamgosar
- Neural Network World
- 2016

## :page_with_curl: Abstract
> Multi-Layer Perceptron Neural Networks (MLP NNs) are the commonly used NNs for target classication. They purposes not only in simulated environments, but also in actual situations. Training such NNs has signicant importance in a way that many researchers have been attracted to this eld recently. Conventional gradient descent and recursive method has long been used to train NNs. Improper classication accuracy, slow convergence speed and trapping in local minimums are disadvantages of the traditional methods. In order to overcome these issues, in recent years heuristic and meta-heuristic algorithms are widely used. This paper uses Gray Wolf Optimization (GWO) algorithm for training the NN. This algorithm is inspired by lifestyle and hunting method of GWs. GWO has a superior ability to solve the high-dimension problems, so we try to classify the Sonar dataset using this algorithm. To test the proposed method, this algorithm is compared to Particle Swarm Optimization (PSO) algorithm, Gravitational Search Algorithm (GSA) and the hybrid algorithm (i.e. PSOGSA) using three sets of data. Measured metrics are convergence speed, the possibility of trapping in local
minimum and classication accuracy. The results show that the proposed algorithm in most cases provides better or comparable performance compared to the other mentioned algorithms.

## :mag_right: 어떤 논문인지 소개해주세요.
- Neural network의 가중치를 GWO로 최적화하는 것에 대해서 다룹니다.

## :key: 핵심 키워드를 적어주세요.
- Classification, Sonar, Multi-Layer Perceptron Neural Network, Grey Wolf Optimization, Particle Swarm Optimization, Gravitational Search Algorithm

## :paperclip: URL
- https://pdfs.semanticscholar.org/8191/270c9239bd7d61f3150e4c61f177cc003d86.pdf
